### PR TITLE
formatting 6/n: metrics

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

apply yapf  to metrics

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: pytorch_lightning/metrics/classification/auroc.py [hunk 1]
```diff
@@ -156,12 +160,4 @@ def compute(self) -> torch.Tensor:
...
-        return _auroc_compute(
-            preds,
-            target,
-            self.mode,
-            self.num_classes,
-            self.pos_label,
-            self.average,
-            self.max_fpr
-        )
+        return _auroc_compute(preds, target, self.mode, self.num_classes, self.pos_label, self.average, self.max_fpr,)
```
Comment (akihironitta) [inline pytorch_lightning/metrics/classification/auroc.py:163]: This should be formatted by applying yapf again.
```suggestion
        return _auroc_compute(preds, target, self.mode, self.num_classes, self.pos_label, self.average, self.max_fpr,)
```

File: pytorch_lightning/metrics/compositional.py [hunk 2]
```diff
@@ -85,8 +85,8 @@ def persistent(self, mode: bool = False):
 
     def __repr__(self):
         repr_str = (
-            self.__class__.__name__
-            + f"(\n  {self.op.__name__}(\n    {repr(self.metric_a)},\n    {repr(self.metric_b)}\n  )\n)"
+            self.__class__.__name__ +
+            f"(\n  {self.op.__name__}(\n    {repr(self.metric_a)},\n    {repr(self.metric_b)}\n  )\n)"
```
Comment (akihironitta) [inline pytorch_lightning/metrics/compositional.py:89]: In my opinion, it looks better without this change here. We could consider adding `split_before_arithmetic_operator = true` to the yapf config to address this.

yapf docs: https://github.com/google/yapf

**Review by ananthsub — APPROVED** [review]

File: pytorch_lightning/metrics/compositional.py (hunk 2 above)
Comment (Borda) [reply pytorch_lightning/metrics/compositional.py]: good point added :]

File: pytorch_lightning/metrics/classification/auroc.py (hunk 1 above)
Comment (Borda) [reply pytorch_lightning/metrics/classification/auroc.py]: rerun now :]

**Review by SkafteNicki — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Review by gianscarpe — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Pull request merged**